backend/src/common/helpers/mailHelper.ts

```diff
@@ -20,7 +21,7 @@ export const sendEmail = async (msg: IEmailMsg) => {
       console.log('Email sent');
     })
     .catch((error: any) => {
-      throw new CustomError(error, 400);
+      throw new CustomError(error, HTTP_STATUS_ERROR_BAD_REQUEST);
```

Comment:
It's better to add enum for http statuses instead of using constants

Reply:
Solved